gcp/appengine/frontend3/src/templates/404.html

```diff
@@ -12,6 +12,27 @@
 {% block top_bar %} {% endblock %}
 
 {% block content %}
+{% if vuln_id %}
+<div class="mdc-layout-grid not-found-page">
+    <div class="mdc-layout-grid__inner">
+        <div class="mdc-layout-grid__cell--span-12 text-info">
+            <h2 class="heading">{{ vuln_id }} Not Found!</h2>
+            <p class="description">
+            {% if has_importfindings %}
+            The record has not been successfully imported from its source<br/>
```

Comment:
Do we plan to surface the actual errors? Or point to where a source provider can see what went wrong?

Reply:
> Do we plan to surface the actual errors? Or point to where a source provider can see what went wrong?

Yes, that's https://github.com/google/osv.dev/issues/2189